src/components/overview/sub-Components/BasicDetails.jsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 
+const productNameStyle = {
+  fontSize: '32pt',
+  color: 'black'
+};
+
 const BasicDetails = function(props) {
   //conditional logic for  a sale
   let prices;
   if (props.state.selectedStyle.selectedReducedPrice > 0) {
     prices = (
       <div>
-        <div>
-          <strike>${props.state.basicDetails.price}</strike>
-        </div>
+        <strike style={{ float: 'left', display: 'inline', width: '10%' }}>
+          ${props.state.basicDetails.price}
+        </strike>
         <div>${props.state.selectedStyle.selectedReducedPrice}</div>
```

Comment:
Good use of having one div below another to have good styling on pricing / discounts.